.github/workflows/ci.yml

```diff
@@ -62,6 +62,14 @@ jobs:
         with:
           cache-on-failure: "true"
 
+      - name: Install cargo-hakari
+        uses: taiki-e/install-action@v2
+        with:
+          tool: cargo-hakari
+
+      - name: Disable hakari
+        run: cargo hakari disable
```

Comment:
Did you observe an impact of this on CI builds (without the other bits)? Just curious to know the delta.

Reply:
I don't think there's an impact, and I'm not sure I'd expect one as I believe that our test will be doing feature unification anyway?